### PR TITLE
[Azure] [OpenAI] Enhance Logs Dashboard

### PR DESCRIPTION
**Pull request by lucian-ioan**
## Proposed commit message

WIP, enhancing Azure OpenAI integration dashboard in preparation for GA.

Improvements:
1. Controls: Model Deployment Name, Model Name, Model Version
2. Lens (by title): Number of Requests, Total Average Latency, Response Codes, Response Codes over time, Average Latency, Request and Response Length.

Unfortunately Azure masks the last octet of the IP so we can't get exact data about specific users.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- 

## Screenshots

Before:
before

After:
dashboard

### CODE REVIEW

File: packages/azure_openai/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.0"
+  changes:
+    - description: Enhance Logs Dashboard.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```
Comment (muthu-mps) [inline packages/azure_openai/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/9968
```

**Comment by muthu-mps** [comment]
Adding more control filter is more useful but we need to also verify how the model details are captured for different logs here.

**Controls**: `Model Deployment Name, Model Name, Model Version`

#### Audit

- The audit log will not have the model specific information. When we apply any of the control above this data will be blank in the dashboard. This may not impact since user will be aware that the audit events may not relate to the model.
- Lets check what happens with the Request Response logs.

#### Request Response
- This logs comes in two different format.
- The request response logs add the model fields only for the successful requests. Not all the error requests logs the model fields.
- When filtering based on the model fields this will show only the successful request and you won't be able to see the failed requests. Requests with 400 or 404 errors will not be listed.

@lucian-ioan - What do you think, If you consider this filters adds more value then lets move them to the second level after the resource controls.

**Comment by lucian-ioan** [comment]
@muthu I believe the control groups for model information make sense even if in some cases the lenses will be empty. Users might want to benchmark different types of models against each other.
 
I agree that the model controls should come after the subscription ones, fixed that.

**Review by muthu-mps — APPROVED** [review]
Please add the dashboard image to the /img directory. Otherwise the change looks good!

**Pull request merged**